src/stravalib/tests/integration/test_client.py

```diff
@@ -49,6 +50,32 @@ def default_request_params():
     return default_request_params
 
 
+@pytest.fixture
+def raw_exchange_response():
+    """Expected response from the protocol exchange_code_for_token method."""
+    return (
+        {
+            "access_token": "123456",
+            "refresh_token": "789sdf987",
+            "expires_at": 1732417459,
+        },
+        {
+            "id": 10295934,
+            "username": "foo_bar",
+            "resource_state": 2,
+            "firstname": "Foo",
+            "lastname": "Bar",
+            "bio": "A bio",
+            "city": "City",
+            "state": "State",
+            "country": "Country",
+            "sex": "F",
+            "premium": True,
+            "summit": True,
+        },
+    )
+
+
 def test_get_athlete(mock_strava_api, client):
```

Comment:
These tests should fully cover this method. oddly code cov isn't updating or recognizing this. but they should test all three scenarios - athlete return missing as strava removes it, athlete return as expected, no athlete data return (default behavior)